docs/astromon.rst

```diff
@@ -0,0 +1,103 @@
+Astromon Components
+===================
+
+Astromon is a package to monitor the astrometric accuracy of the Chandra X-ray observatory.
+
+Database
+--------
+
+The main data product of Astromon is a database that contains the following tables:
+
+* Observations (astromon_obs). Observation info from OBSPARS.
+* X-ray sources (astromon_xray_src). Point-sources detected in Chandra observations.
+* Catalog sources (astromon_cat_src). Optical/Radio catalog entries in the vicinity of the X-ray
+  sources
+* Cross-matches (astromon_xcorr). Pairs of X-ray and catalog sources, where there is at most one
+  catalog source associated with a given detected X-ray source. 
+
+Astromon data can be accessed through the astromon.db module using convenience functions that
+return the tables as an astropy.table.Table::
+
+    from astromon import db
+    matches = db.get_cross_matches()
+    observations = db.get('astromon_obs')
+    xray_src = db.get('astromon_xray_src')
+    cat_src = db.get('astromon_cat_src')
+    xcorr = db.get('astromon_xcorr')
+
+NOTE: One can modify the behavior of astromon.db and specify a different database file by defining
+the ASTROMON_FILE environmental variable.
+
+A more elaborate example to do some analysis, correlating several MSIDs with astromon data, and
```

Comment:
What I would also like to see is an example of doing a non-standard cross match, e.g. with a lower SNR threshold than standard and for sources within 1 arcmin. 

I think this might require an update to getting cross matches, since the current `get_cross_matches` function relies on the pre-computed "standard join" `astromon_xcorr` table instead of doing all the joins dynamically and then allowing filtering after the joins. This was my approach in the original `astropy-play-API` notebook with the `get_xcorr` and `filter_xcorr` functions.

It's also possible I just missed how to do this with the existing code.